hybrasyl/World.cs

```diff
@@ -675,6 +682,38 @@ private bool LoadData()
                 WorldData.SetWithIndex(messageboard.Name, messageboard, messageboard.Id);
             }
 
+            // Load user uuid reference and initialize
+            foreach (var key in server.Keys(pattern: "User*"))
+            {
+                GameLog.InfoFormat("Loading Uuid references for {0}", key);
+                var user = DatastoreConnection.GetDatabase().Get<User>(key);
+                var name = key.ToString().Split(':')[1];
+                if (name == string.Empty)
+                {
+                    GameLog.Warning("Potentially corrupt user data in Redis; ignoring");
+                    continue;
+                }
+                var uuidKey = $"Hybrasyl.UuidReference:{name}";
+                var uuidRef = DatastoreConnection.GetDatabase().Get<UuidReference>(uuidKey);
+                if (uuidRef == null)
+                {
+                    GameLog.WarningFormat("No Uuid Reference found for {0}, rebuilding", user.Name);
+                    uuidRef = new UuidReference(name);
+                    uuidRef.AccountUuid = user.AccountUuid;
+                    uuidRef.UserUuid = user.Uuid;
+
+                    //sanity check to make sure user has this object
+                    if(!WorldData.ContainsKey<ParcelStore>($"Hybrasyl.ParcelStore:{uuidRef.UserUuid}"))
+                    {
+                        var parcelStore = new ParcelStore(uuidRef.UserUuid);
+                        WorldData.Set(parcelStore.OwnerUuid, parcelStore);
+                        parcelStore.Save();
+                    }
+                }
+                WorldData.SetWithIndex(user.Name, uuidRef, user.Name);

```

Comment:
It seems like this should be `(user.Name, uuidRef, user.Uuid)`

Reply:
will adjust